### PR TITLE
Innovative comments, no longer use isset, and trademarking enhanced.

### PR DESCRIPTION
**Pull request by y04nqt**
Just some general trademarking for insurance of credit where credit is due. Also added some good documentation to help the cause. Had to put a call to action to add more as a good TODO/Reminder for any PHPainfreers out there!

### CODE REVIEW

File: htdocs/index.php [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <?php
-
+	// Here, we become infinite.
```
Comment (y04nqt) [inline htdocs/index.php:2]: This is what I've been living for this entire time and now I can do it forever. Thanks PHPainfree!

File: includes/Painfree.php [hunk 2]
```diff
@@ -94,7 +94,7 @@ public function __construct($options) {
 		list($root_path,$junk) = explode($this->options['LogicFolder'], __FILE__);
 		$this->Root = $root_path;
 
-		$this->route = isset($_REQUEST[$this->options['RouteParameter']]) ?
+		$this->route = !empty($_REQUEST[$this->options['RouteParameter']]) ?
```
Comment (y04nqt) [inline includes/Painfree.php:97]: quality code 👍🏻

File: includes/core/DBI.php [hunk 3]
```diff
@@ -17,7 +17,7 @@ public function __construct($db_config) {
 			$this->Connections = $db_config;
 			
 			foreach ( $this->Connections as $server => $Conn ) {
-				$dbd = isset($Conn['type']) ? strtolower($Conn['type']) : 'mysql';
+				$dbd = !empty($Conn['type']) ? strtolower($Conn['type']) : 'mysql';
```
Comment (y04nqt) [inline includes/core/DBI.php:20]: super nice addition here

File: templates/debug.php [hunk 4]
```diff
@@ -15,7 +15,7 @@
 	
 	$Painfree->debug('$PainfreeConfig', $PainfreeConfig);
 
-	$Painfree->debug('EXAMPLE', 'You can basically pass anything to $Painfree->debug().');
+	$Painfree->debug('EXAMPLE', 'You can basically pass anything to $Painfree->debug().™');
```
Comment (y04nqt) [inline templates/debug.php:18]: NITPICK: should we put the TM before or after the period? I'm conflicted.

File: templates/generic.php [hunk 5]
```diff
@@ -2,7 +2,7 @@
 	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
 	<head>
-		<title><?php echo $Generic->title(); ?></title>
+		<title><?php echo $Generic->title(); ?>™</title>
```
Comment (y04nqt) [inline templates/generic.php:5]: THIS IS INNOVATION

File: templates/generic.php [hunk 6]
```diff
@@ -29,9 +29,17 @@
 			and put it in templates and have your BaseView automatically load it from <code>templates/views/</code>.
 		</p>
 		<p>
-			But that's just me... do what you want.
+			But that's just me... do what you want. I do command you to exclusively use PHPainfree from here on out.
+			It's in the contact, Mike.
```
Comment (y04nqt) [inline templates/generic.php:33]:
```suggestion
			It's in the contract, Mike.
```

File: templates/generic.php [hunk 7]
```diff
@@ -29,9 +29,17 @@
...
 		</p>
-		<h2>Debugger</h2>
+		<p>
+			We also suggest that you create a documentation site as you progress along your new found love project.
+			Please ensure it is accessible to the community of PHPainfree. We are watching.
+		</p>
+		<p>
+			This is an open source project that needs your help to make it truly, <code>PAINFREE.</code>™
```
Comment (y04nqt) [inline templates/generic.php:40]: True! 🙌🏻

File: templates/view/main.php [hunk 8]
```diff
@@ -0,0 +1,3 @@
+<?php 
+	echo "What's up, what's up, what's up PaaaaaaaaainFreeeeeeeeeeee!™ WOOOOOOOOOOO!";
```
Comment (y04nqt) [inline templates/view/main.php:2]: Painfree is my FAVORITE. EoD.

File: templates/generic.php (hunk 6 above)
Comment (y04nqt) [reply templates/generic.php:33]: nice catch! 🎣

File: htdocs/index.php (hunk 1 above)
Comment (iethree) [reply htdocs/index.php:2]: @y04nqt right now

raw

**Comment by y04nqt** [comment]
GH won't let me approve this, but I approve this. GTG, ready to soar 🚀 🚀 🚀

File: htdocs/index.php (hunk 1 above)
Comment (y04nqt) [reply htdocs/index.php:2]: youreright

**Pull request closed without merging**